web/content/versions.md

```diff
@@ -1,2 +1,3 @@
-[Nightly](/nightly.html)
-[2.3](/2.3.html)
+[Foreman and Katello Nightly](/nightly.html)
+[Foreman 2.3 and Katello 3.18](/2.3.html)
+[Foreman 2.4 and Katello 4.0](/2.4.html)
```

Comment:
Since Katello docs 4.0 are now linked and the landing page is this one, I thought it would be good idea to present both versions so Katello users do not need to figure out the version.